rasa/core/training/dsl.py

```diff
@@ -504,6 +504,10 @@ def add_event(self, event_name, parameters):
             action_as_message = MarkdownReader().parse_e2e_training_example(event_name)
             parameters["name"] = action_as_message.text
             parameters["message"] = action_as_message
```

Comment:
If we don't use the `message` as the storage of features (as we will rather get them on the fly), should we remove it?

Reply:
Yes, eventually. I left these parts because I didn't want to break your current solution. Happy to help you cleaning up the featurizer and removing some of the hacks.

Reply:
You're right :) I'd rather we leave it working for and remove once featurizers are fixed.